src/component/registry/types.ts

```diff
@@ -244,7 +260,11 @@ export type StyleWithOpacity = {
    * - If the set value is less than 0, the used value is 0.
    * - If the set value is greater than 100, the used value is 100.
    *
-   * The special `default` value is to use the color defined in the BPMN element default style
+   * **NOTE**: `opacity` does not apply to the font style.
+   *
+   * Notes about the `default` special keyword:
+   * - It is to use the opacity defined in the BPMN element default style.
+   * - It can be used when the style is first updated and then needs to be reset to its initial value.
```

Comment:
**suggestion:** 
```suggestion
   * **NOTE**: The `opacity` property does not affect the font style.
   * Additional information about the `default` keyword:
   * - It is used to apply the opacity defined in the default style of the BPMN element.
   * - It can be used when updating the style initially and then resetting it to its original value.
```

Reply:
:warning: We cannot write `` `opacity` does not apply to the font style.`` here, because this type is use in the type `Font`, and the opacity works in this level.